### ISSUE TITLE
Package Manager Error

### ISSUE THREAD

**Issue by lapulgaatomica10** [microsoft/BotFramework-Composer#8631]
## Describe the bug

![image](https://user-images.githubusercontent.com/90028529/131967395-35b38f2a-27fa-4ec5-9586-ff7aceba5604.png)

## Version

Version:  2.1.0-nightly.266659.7c14b7a
Electron: 8.2.4
Chrome: 80.0.3987.165
NodeJS: 12.13.0
V8: 8.0.426.27-electron.0

## Browser

- [ ] Electron distribution
- [x] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

**Comment by benbrown** [comment]
@lapulgaatomica10 Did you install or uninstall any components, or does this happen with a brand new bot?

Can you reproduce the problem with a brand new empty bot?

**Comment by lapulgaatomica10** [comment]
@benbrown yes I tried install and uninstall the composer and after that I tried it on a brand new bot and it is still the same. Should I  uninstall and install the pre-requesites of composer and try it again?

**Comment by scottpulver** [comment]
Im getting same issue with brand new QnA Maker template. Navigating to the Package Manager gives this error.

Could not load component list:Error https://schemas.botframework.com/schemas/component/v1.0/component.schema: self signed certificate in certificate chain Error Microsoft.QnAMakerDialog: Cannot extend component because it is not included Error Microsoft.QnAMakerDialog: Cannot set property 'const' of undefined *** Could not merge components ***

**Comment by compulim** [comment]
@benbrown Ping. Do you think it's an installation issue?

**Comment by lapulgaatomica10** [comment]
Hi @benbrown how can I know if its network problems? I mean where should I check?

**Comment by benbrown** [comment]
@lapulgaatomica10 Are you on a restricted network, or do you access the internet normally through a proxy? The message about the `self signed certificate in certificate chain` might indicate that the request for the schema file is being blocked and redirected and thus failing the SSL check.

**Comment by henrygarciav7** [comment]
I have the same issue, I have verified and we have implemented Netscope in our company. I need de the URL's that should be added to the exception rules. Please help me wiht these.

**Comment by mrw137** [comment]
I have exactly the same issue. Is there a solution?
